lib/reek/context/code_context.rb

```diff
@@ -51,6 +54,30 @@ class CodeContext
       def initialize(context, exp)
         @context = context
         @exp     = exp
+
+        @num_statements = 0
+        @refs = AST::ObjectRefs.new
+      end
+
+      def count_statements(num)
+        @num_statements += num
+      end
+
+      def record_call_to(exp)
+        receiver, meth = exp[1..2]
```

Comment:
Hmm, you're adding to our technical debt here by using deprecated methods: https://github.com/troessner/reek/issues/546
I don't think we should do this...